neutron-org/neutron-dao
# feat: overrule pre-proposal module #NTRN-303

This PR is to add
1. Pre-proposal module to build an overrule proposals for timelocked decisions of subdaos
2. It modifies the integration script for timelocks by making the overrule not manually from admin account but with the overrule proposal made by main governance

How to test:
1. Pull [the branch of this PR](https://github.com/neutron-org/neutron/pull/120) from main Neutron repo
1. run `make kill-dev && make install && make init` in `neutron` dir
1. run `./test_subdao_proposal.sh` in `neutron-dao`

(also check if the version of `cwd_pre_propose_overrule.wasm` in `neutron/contracts` corresponds to the current repo code by building the release version and checking the hash)
